edit-post/hooks/meta/index.js

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
I'd expect this file to live in `editor.hooks` for now. I understand that it's post related but I believe the refactoring to extract all post related stuff should be done in one step instead of introducing inconsistency.